src/DependencyInjection/SentryExtension.php

```
@@ -280,7 +279,6 @@ private function configureErrorListenerIntegration(array $integrations, bool $re
                 [
                     'ignore_exceptions' => [
                         FatalError::class,
-                        FatalErrorException::class,
```

Are you sure we can stop ignoring this exception? In Symfony `4.4` it is still present

---

Mh maybe you're right, I'm not sure how 4.4 handles this deprecation. Let's keep it safe, I'll revert this piece.